content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,199 @@
 
-To start, download the example project. The project includes the configuration that you need to complete this guide.
-
-### Steps
-
-1. Open a terminal window and navigate to a location of your choice. Use the following commands to create a directory for the project and download the example code:
-
-   ```terminal wrap
-   mkdir prisma-aws-lambda
-   cd prisma-aws-lambda
-   curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=3 prisma-examples-latest/deployment-platforms/aws-lambda/
-   ```
-
-2. Install the dependencies:
-
-   ```terminal
-   npm install
-   ```
-
-<!-- tar strip folder is a concatenation of the REPOSITORY-BRANCH/REF, e.g. prisma-examples-latest -->
-
-### Result
-
-Your directory contains the following files:
-
-```terminal no-copy
-README.md
-handlers
-node_modules
-package-lock.json
-package.json
-prisma
-schema.sql
-serverless.yml
+```code file=serverless.yml no-copy
+package:
+  patterns:
+    - '!node_modules/.prisma/client/libquery_engine-*'
+    - 'node_modules/.prisma/client/libquery_engine-rhel-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - '!node_modules/@prisma/engines/**'
 ```
 
-## 2. Set the DATABASE_URL in <inlinecode>.env</inlinecode>
-
-The Lambda functions need the `DATABASE_URL` environment variable to access the database. You must define an `.env` file. The preconfigured `serverless-dotenv-plugin` uses this file to inject into the function runtime.
-
-The repository contains an `.env.example` example file to help with this.
-
-### Steps
-
-1. Enter the following command to copy `.env.example` to a new file, `.env`:
-
-   ```terminal
-   cp .env.example .env
-   ```
-
-2. Open `.env` and set the `DATABASE_URL` to your database credentials. The format is as follows:
-
-   ```file
-   DATABASE_URL="postgres://__USER__:__PASSWORD__@__HOST__/__DATABASE__"
-   ```
+However, if you use `serverless-webpack`, then you cannot use this method. See [Deployment with serverless webpack](#deployment-with-serverless-webpack) for details.
 
-   Replace the uppercase placeholders with your database credentials. For example:
+### Deployment with <inlinecode>serverless-webpack</inlinecode>
 
-   ```file
-   DATABASE_URL="postgres://janedoe:randompassword@yourpostgres.supabase.co:6543/yourdbname"
-   ```
+If you use `serverless-webpack`, then you can use the [serverless-webpack-prisma](https://github.com/danieluhm2004/serverless-webpack-prisma) plugin without any additional configuration. `serverless-webpack-prisma` does the following:
 
-   > **Note:** If your connection URL contains any special characters, then you must [percentage encode them](/reference/database-reference/connection-urls#special-characters).
+1. Copies your `schema.prisma` with the Webpack plugin.
+2. Runs the `prisma generate` command. This means that there is no need to add the command to the webpack options separately.
+3. Packages the correct Prisma engine binary. This can save more than 40mb of capacity.
```

Comment:
Second sentence here is super weird, as I am not sure you can "save capacity", and 2 because the number is most probably outdated and wrong.